docs/Development.md

```diff
@@ -113,3 +113,50 @@ These called functions are implemented in the runtime, `src/CHIPBindings.cc`; th
 `__hipRegisterVar`: this registers global variables, again connecting a host pointer with a variable name.
 
 there is an additional `__hipUnregisterFatBinary()` called after main() returns; in CHIP-SPV, this calls `CHIPUninitialize()` once the number of loaded modules becomes zero.
+
+
+# Release management
+
+For each release, a release manager is assigned. Release manager is responsible
+for creating and requesting testers from different platforms. After a release
+candidate round with success reports and no failure reports, a release is published.
+
+A checklist of things to do for a release:
+
+* Check that CHANGES has the most interesting updates since last release.
+  Add missing notable changes from git log.
+
+* Update the docs/Features.md with any changes.
+
+* Create a single commit in main branch: change the version to the
+  release one (without -pre), in all relevant places (CHANGES, docs,
+  CMakeLists.txt, etc); update the libCHIP.so version (if required);
+  check that supported LLVM versions are correct.
+  Create the release branch from this commit and push it to github.
```

Comment:
Single commit in the main branch? Normally main branches are protected so I think it would make sense to make a release branch that contains these changes.
@franz